frontend/packages/data-portal/app/components/Deposition/MethodLinks/MethodLinksMetadataTable.tsx

```diff
@@ -137,9 +137,13 @@ export function MethodLinksMetadataTable({
         .map((methodData, i) => (
           <div className="flex flex-col gap-sds-xl">
             <MethodSummarySection
-              label={t('methodCount', {
-                value: startCase(converter.toWords(i + 1)),
-              })}
+              label={
+                deposition.annotation_methods.length === 1
```

Comment:
nit - comment why `1` is the only value that's ignored

Reply:
design wants if theres only one method for there not to be a "Method One" so we only ignore 1; if there's 0 it wont be executed at all and otherwise all values should display the header